Makefile

```diff
@@ -386,7 +387,8 @@ create-cluster: $(CLUSTERCTL) $(KUSTOMIZE) $(ENVSUBST) ## Create a development K
 	  sed "s|\$${OPENSTACK_CLOUD_PROVIDER_CONF_B64}|$(OPENSTACK_CLOUD_PROVIDER_CONF_B64)|" | \
 	  sed "s|\$${OPENSTACK_CLOUD_CACERT_B64}|$(OPENSTACK_CLOUD_CACERT_B64)|" | \
 	  sed "s|\$${KUBERNETES_VERSION}|$(KUBERNETES_VERSION)|" | \
-	  sed "s|\$${CLUSTER_NAME}|$(CLUSTER_NAME)|"  \
+	  sed "s|\$${CLUSTER_NAME}|$(CLUSTER_NAME)|"  | \
+	  sed "s|\$${CI_VERSION}|$(CI_VERSION)|" \
```

Comment:
This change breaks our e2e tests as stuff like `${CI_VERSION//+/%2B}` is not properly replaced. I don't unterstand why you have to set the CI_VERSION locally?

Reply:
I also wouldn't recommend create-cluster for usage outside of our e2e tests. As it always patches the node bootstrapping to use artifacts from Google Cloud storage (CI). Which is slower then just installing CAPO the regular way

Reply:
I guess this PR fixes it because it probably breaks the node bootstrapping? (I guess the if in the e2e-conformance_patch.yaml.tpl is not executed anymore)

Reply:
ok, I will manually patch my system
the reason is my bootstrap is blocked by some CI scripts locally
so I think it's worthy to avoid use it

and I think the logic is same , I still keep the CI_VERSION as its original value..

Reply:
@jichenjc is this PR still active?